sdks/python/apache_beam/transforms/external.py

```diff
@@ -286,7 +286,8 @@ def expand(self, pvalueish):
     pipeline = (
         next(iter(self._inputs.values())).pipeline
         if self._inputs else pvalueish.pipeline)
-    context = pipeline_context.PipelineContext()
+    context = pipeline_context.PipelineContext(
+        component_id_context=pipeline._component_id_context)
```

Comment:
Based on this use, maybe `_component_id_context` isn't private at all? And maybe it is a `component_id_map` or some such. It isn't "context" in the sense that it isn't "the stuff surrounding the thing you are interested in".

Reply:
Good point, renamed it to `component_id_map` everywhere